SingularityUI/app/components/webhooks/Webhooks.cjsx

```diff
@@ -0,0 +1,215 @@
+React = require 'react'
+ReactDOM = require 'react-dom'
+Utils = require '../../utils'
+Table = require '../common/Table'
+PlainText = require '../common/atomicDisplayItems/PlainText'
+TimeStamp = require '../common/atomicDisplayItems/TimeStamp'
+Link = require '../common/atomicDisplayItems/Link'
+Glyphicon = require '../common/atomicDisplayItems/Glyphicon'
+NewWebhookForm = require './NewWebhookForm'
+vex = require 'vex'
+
+Webhooks = React.createClass
+
+    defaultRowsPerPage: 10
+
+    rowsPerPageChoices: [10, 20, 30, 40]
```

Comment:
webhooks is generally going to be a very small set of items, I'd be surprised if any users even had enough to fill a page of 10. Maybe just limit this to 10, 20 so we aren't presenting options with way more than we need?